src/content/guides/processing-centrapay-transactions.mdoc

```diff
@@ -0,0 +1,95 @@
+---
+title: Processing a Centrapay Transaction
+description: How to process a Centrapay transaction.
+nav:
+  path: Merchant Integrations
+  order: 2
+---
+
+## Compatible QR Payment Methods
+Currently, users can scan Centrapay QR codes through multiple methods, including:
+{% wrapper class="flex flex-wrap gap-6 justify-start not-prose [&>p]:w-20 [&>p]:text-center [&>p]:font-normal [&>p]:text-[10px] [&>p]:leading-[1.5] [&>p]:tracking-[-0.025em] [&>p]:font-sans [&>p>img]:mx-auto [&>p>img]:mb-2 [&>p>img]:!w-8 [&>p>img]:!h-auto [&>p>img]:!max-w-none" %}
```

Comment:
@alexBinns these classes are very suspicious - can the class `[&>p>img]:!h-auto` not be put on the imageGrid?